scripts/channelcommands.js

```diff
@@ -1,4 +1,4 @@
-exports.handleCommand = function (src, command, commandData, tar, channel) {
+﻿exports.handleCommand = function (src, command, commandData, tar, channel) {

```

Comment:
This character may get silently deleted by one or more browsers.<br>'exports' is not defined.